### PR TITLE
Add contracts verification script

### PR DESCRIPTION
**Pull request by zguesmi**

### CODE REVIEW

File: scripts/voucherHubUtils.ts [hunk 1]
```diff
@@ -94,6 +94,6 @@ export async function getExpectedVoucherProxyCodeHash(voucherBeaconAddress: stri
          * Note: Look very carefully before updating this value to avoid messing with
          * existing vouchers already deployed in production.
          */
-        return '0x386d2735c0e7b2144a00e43bc5451acd8063573f30f1edc7b8e8891039cbd84f';
+        return '0xba1aec6240a4c35e4711b8ed9013b7204c7d65e1caa6f364b3793946146e8aaf';
```
Comment (gfournieriExec) [inline scripts/voucherHubUtils.ts:97]: Why this change ?

**Review by gfournieriExec — APPROVED** [review]

File: scripts/voucherHubUtils.ts (hunk 1 above)
Comment (zguesmi) [reply scripts/voucherHubUtils.ts:97]: The CI would fail without this.

Comment (gfournieriExec) [reply scripts/voucherHubUtils.ts:97]: Mmh ok I see, if it's fails then we should change it ! But I don't get what are the root cause of this change. That was due to a modification, evmVersion: 'berlin', ?
